### PR TITLE
Adding count4net.IRateCounter to exceptions list

### PR DESCRIPTION
**Pull request by csilcock**
Adding the exception @saquibmian and I discussed for `count4net.IRateCounter`

### CODE REVIEW

File: src/D2L.CodeStyle.Analyzers/Common/MutabilityInspector.cs [hunk 1]
```diff
@@ -20,6 +20,7 @@ public sealed class MutabilityInspector {
 		/// A list of known immutable types
 		/// </summary>
 		private static readonly ImmutableHashSet<string> KnownImmutableTypes = new HashSet<string> {
+			"count4net.IRateCounter",
```
Comment (saquibmian) [inline src/D2L.CodeStyle.Analyzers/Common/MutabilityInspector.cs]: Is `IStatCounter` cool too?

File: src/D2L.CodeStyle.Analyzers/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -10,7 +10,7 @@
 
 [assembly: ComVisible( false )]
 
-[assembly: AssemblyVersion( "0.9.0.0" )]
-[assembly: AssemblyFileVersion( "0.9.0.0" )]
+[assembly: AssemblyVersion( "0.9.1.0" )]
```
Comment (saquibmian) [inline src/D2L.CodeStyle.Analyzers/Properties/AssemblyInfo.cs]: I'm confused why you don't get a merge conflict here. `0.9.1.0` was taken recently: https://github.com/Brightspace/D2L.CodeStyle/blob/master/src/D2L.CodeStyle.Analyzers/Properties/AssemblyInfo.cs#L13

File: src/D2L.CodeStyle.Analyzers/Common/MutabilityInspector.cs (hunk 1 above)
Comment (csilcock) [inline src/D2L.CodeStyle.Analyzers/Common/MutabilityInspector.cs]: Why yes, it is!

File: src/D2L.CodeStyle.Analyzers/Properties/AssemblyInfo.cs (hunk 2 above)
Comment (csilcock) [inline src/D2L.CodeStyle.Analyzers/Properties/AssemblyInfo.cs]: That is odd - I pulled this repo for the first time today, and it had it at 0.9.0.0 locally prior to my work... I'll bump it again to 0.9.2.0

**Comment by csilcock** [comment]
Thanks for fixing the conflict, @saquibmian

**Pull request merged**